src/twin/system_info.rs

```diff
@@ -118,4 +204,112 @@ impl SystemInfo {
         .await
         .context("report: send")
     }
+
+    async fn metrics_handler(&mut self) -> Result<()> {
+        let time = match time::OffsetDateTime::now_utc().format(&Rfc3339) {
+            Ok(time) => time,
+            Err(e) => {
+                warn!("timestamp could not be generated: {e}");
+                return Ok(());
+            }
```

Comment:
Can this happen on RPi without RTC(?). If not I would prefer to fail. Otherwise I would log an error and whitelist.
Further I would change to:
```
let Ok(time) = time::OffsetDateTime::now_utc().format(&Rfc3339) else {
      error!("metrics_handler: timestamp could not be generated");
      return Ok(());
```

Reply:
This is independent from having a RTC. It should never fail.

Reply:
changed to bail!